.github/workflows/ci.yml

```diff
@@ -12,15 +12,20 @@ jobs:
           fetch-depth: 0
       - uses: ohpensource/ensure-conventional-commits-gh-action@main
         name: Ensure conventional commits
-      - uses: ohpensource/ensure-commits-have-ticket-gh-action@main
+      - uses: ohpen/ensure-commits-have-ticket-gh-action@main
```

Comment:
@cangulomascarell: maybe we can migrate also this repository to Ohpen organization (and maybe we can do it for all repositories)